doc/classes/SceneTree.xml

```diff
@@ -108,7 +108,7 @@
 		<method name="get_frame" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns how many frames have been processed, since the application started. This is [i]not[/i] a measurement of elapsed time.
+				Returns how many physics_process steps have been processed, since the application started. This is [i]not[/i] a measurement of elapsed time.
```

Comment:
```suggestion
				Returns how many [code]physics_process[/code] steps have been processed, since the application started. This is [i]not[/i] a measurement of elapsed time.
```